startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
umm that's not a daemon. That code you linked is tied to Chaos' process. Once Chaos' process is replaced, that code is no longer running.